.env.example

```diff
@@ -8,3 +8,9 @@ TALK_PATH=spreed
 # Defaults to "*" which might be unsafe during development - you can set it to allow only your hosts
 # Example: https://nextcloud.local wss://signaling.nextcloud.local wss://notify_push.nextcloud.local
 NEXTCLOUD_DEV_SERVER_HOSTS=*
+
+# Application channel - stable or preview
```

Comment:
would using "beta" be problematic? Then the naming would match the one on server